services/core-web/src/components/mine/MineHeader.js

```diff
@@ -119,6 +119,8 @@ export class MineHeader extends Component {
       exemption_fee_status_code: mine.exemption_fee_status_code,
       exemption_fee_status_note: mine.exemption_fee_status_note,
       government_agency_type_code: mine.government_agency_type_code,
+      number_of_mine_employees: mine.number_of_mine_employees ? mine.number_of_mine_employees : null,
```

Comment:
If either of these is 0, it will be set to null because 0 is falsey in JS. Is this correct?
Statement could also be reduced to `mine.number_of_mine_employees ?? null`

Reply:
Good one. Changed. Thanks :)